dspace/config/crosswalks/oai/xoai.xml

```diff
@@ -90,6 +91,23 @@
                 This contexts complies with Openaire Guidelines for Literature Repositories v4.0.
             </Description>
         </Context>
+        <!--
+            RIOXX Guidelines v3.0 Final:
+            - https://www.rioxx.net/profiles/v3-0-final
+
+         -->
+        <Context baseurl="rioxx" name="Rioxx Context">
+            <!-- Date format, field prefixes, etc are ensured by the transformer -->
+            <Transformer ref="rioxxTransformer"/>
+            <Filter ref="rioxxFilter"/>
+            <!-- Metadata Formats -->
+            <Format ref="oaidc"/>
+            <Format ref="rioxx"/>
+            <Format ref="qdc"/>
+            <Description>
+                This contexts complies with RIOXX rules.
```

Comment:
Do we need to specify that this is RIOXX v3?  Also, if I'm not mistaken, aren't they calling this "Rioxx" now instead of "RIOXX"? https://rioxx.net/

Maybe this description should say something like:
```
This context complies with the Rioxx metadata profile version 3
```